Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -55,14 +85,12 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager) {
 
 	@Override
 	public double getLeftTotalDistance() {
-		// TODO Auto-generated method stub
-		return 0;
+		return ticksToInches(leftMaster.getSelectedSensorPosition(0));
```

Comment:
We should be doing the inch conversion for each wheel individually rather than using the average factor for returning metrics on individual wheels. This could be especially important for our offboard processing system.

Reply:
Sounds reasonable